src/Components/Books/Book.css

```diff
@@ -1,3 +1,121 @@
-* {
-  margin: 0;
+.bookCard {
+  width: 90%;
+  height: 10.625rem;
+  padding: 20px;
+  margin: auto;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  background-color: rgb(255, 255, 255);
+  border: 1px solid #f0ecec;
+  margin-bottom: 10px;
+}
+
+.bookCard .book_container_left {
+  width: 45%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+}
+
+.category {
+  font-size: 1.2rem;
+  color: #868686;
+}
+
+.book_container_left p.title {
+  font-size: 1.4rem;
+  font-weight: bold;
+  font-family: 'Times New Roman', Times, serif;
+  margin: 5px 0 3px 0;
+}
+
+.author {
+  font-family: 'Times New Roman', Times, serif;
+  color: #0290ff;
+}
+
+.book_container_left .btn_left_container {
+  width: 40%;
+  display: flex;
+  list-style: none;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.book_container_left .btn_left_container .bar {
+  width: 1px;
+  height: 15px;
+  background-color: #e2e2e2;
+}
```

Comment:
- [ ] One of the best practices when writing codes in style sheets is to avoid the repetition of `rules/ declarations`. You can decide to create a class for each colour and apply the color classes as `className` on the `JSX` element you wish to apply it on or you can decide to use CSS variables. Kindly apply this change when declaring the following:

- `#e2e2e2`
- `#868686`
- `#0290ff`
- `#fff`
- The `font-families` that you will use:
  - `Roboto`
  - Montserrat

#### Kindly implement this change on all stylesheets used in this project. :'